### ISSUE TITLE
How To Compute Perplexity of Generated Sequences + Interpretation of ESM-3's sequence logits values

### ISSUE THREAD

**Issue by issah-samori** [evolutionaryscale/esm#68]
Hello, 

First off, thank you for open sourcing this amazing project. I generated some sequences and I want to compute the perplexity of the sequences. I was wondering if I could be guided on how to do that.

Also, in an attempt to compute the perplexity, I looked at the sequence logits (shown below):
```
tensor([[[-19.3215, -19.2987, -19.3012,  ..., -19.3543, -19.3293, -19.3383],
         [-17.1530, -17.1291, -17.0036,  ..., -17.0593, -17.1460, -17.0795],
         [-19.6029, -19.5236, -19.5956,  ..., -19.5862, -19.5897, -19.5557],
         ...,
         [-21.7179, -21.7085, -21.6753,  ..., -21.7226, -21.7259, -21.7640],
         [-19.0487, -19.0818, -19.1109,  ..., -19.0751, -18.9670, -18.9925],
         [-19.2255, -19.1504, -19.2251,  ..., -19.1898, -19.1538, -19.3193]]],
       device='cuda:0')
```
The logits have values less than 0 and also some values greater than 1. I was under the impression the logits are supposed to represent probability values. I was wondering how this is possible or what is the interpretation of ESM-3's sequence logits values. Thank you!

**Comment by ebetica** [comment]
logits are unnormalized log probabilities. You can convert them to probabilities via a `softmax` functions.

Feel free to reopen this issue if you have further questions.
